es-core/src/platform.cpp

```diff
@@ -85,5 +85,8 @@ void processQuitMode()
 		touch("/tmp/es-shutdown");
 		runShutdownCommand();
 		break;
+	default:
```

Comment:
Add default case to catch any unhandles values:

/home/devel/temp/EmulationStation_Stable/es-core/src/platform.cpp:72:10: warning: enumeration value '
QUIT' not handled in switch [-Wswitch]
        switch (quitMode)
                ^~~~~~~~
1 warning generated.